app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1439,7 +1462,7 @@ public void onBackPressed() {
         }
         if (atHome) {
             if (backPressedOnce) {
-                finish();
+                finishAffinity();
```

Comment:
@the-dagger But the things is I am not switching to a different activity infact I am closing the application, so how should I set the intent for it?